prog/dftb+/lib_solvation/born.F90

```diff
@@ -374,7 +471,13 @@ subroutine getStress(self, stress)
     @:ASSERT(self%tPeriodic)
     @:ASSERT(self%volume > 0.0_dp)
 
-    stress(:,:) = self%stress / self%volume
+    if (allocated(self%sasaCont)) then
+      call self%sasaCont%getStress(stress)
+    else
+      stress(:, :) = 0.0_dp
+    end if
+
+    stress(:,:) = stress + self%stress / self%volume
```

Comment:
Isn't self%stress actually self%virial at this point? Might consider re-naming, if we're going to tidy all of these up.

Reply:
Usually going for `sigma` but `virial` should work as well.

Reply:
sigma is good.